librz/include/rz_list.h

```diff
@@ -19,7 +19,7 @@ typedef struct rz_list_t {
 	RzListIter *head;
 	RzListIter *tail;
 	RzListFree free;
-	int length;
```

Comment:
`size_t`?

Reply:
i think u32 is bign enough. we will never use 4+ GB of data in these lists.

Reply:
`size_t` is problematic in such cases, since it means we will support different lengths of the lists on different platforms. It isn't good thing. Better to stick to either `ut32` or `ut64`, IMHO.

Reply:
But creating `ut64`-sized list would be a grave mistake anyway.